### DISCUSSION TITLE
MFXTItledPane Documentation not clear

### DISCUSSION THREAD

**Discussion by PLEUTOSEEK** [palexdev/MaterialFX#224] [Archived]
Hello seniors, may I ask why the MFXTitledPane are unavailable to put anything inside it with scene builder? If available, any documentation link available for me to refer? I will be appreciate that if anyone could help me  even a bit. Thanks in advance

**Comment by palexdev** [comment]
1. MFXTitledPane is still experimental (since it's in on the staging branch), features, behavior and documentation are likely to change in the future
2. Unfortunately SceneBuilder is a mediocre tool that privileges JavaFX original controls and leaves no way to custom controls to specify behaviors. So, no, there's no way to tell SceneBuilder to set the pane's content
